File: runtime-testsuite/scripts/csharp/issue-2693/CSharp/ParserBase.cs
using Antlr4.Runtime;
using System.IO;

public abstract class ParserBase : Parser
	{
        public ParserBase(ITokenStream input, TextWriter output, TextWriter errorOutput) : this(input)
        {
        }

        protected ParserBase(ITokenStream input)
   		: base(input)
		{
		}
	}

File: runtime-testsuite/scripts/csharp/issue-2693/Java/ParserBase.java
import org.antlr.v4.runtime.*;

public abstract class ParserBase extends Parser
{
//    public ParserBase(TokenStream input, TextWriter output, TextWriter errorOutput)
//    {
//	super(input);
//    }

    public ParserBase(TokenStream input)
    {
	super(input);
    }
}
